### ISSUE TITLE
Can trigger own functionality when transpiling?

### ISSUE THREAD

**Issue by leoloso** [phabelio/phabel#10]
I have a WordPress plugin, coded in 8.0, to be transpiled to 7.1. The required PHP version must be indicated in the plugin's main file header:

```php
<?php
/*
Requires PHP: 8.0
*/
```

Is it possible to inject some custom functionality, so that when we trigger the transpiling, this header is converted into this?

```php
<?php
/*
Requires PHP: 7.1
*/
```

**Comment by danog** [comment]
Absolutely, I actually already prepared most of the logic needed for custom phabel plugins, I just need to enable the functionality in the composer plugin, will do ASAP.
